src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -3353,7 +3353,7 @@ pub enum Explainee<T: AstInfo> {
     View(T::ItemName),
     MaterializedView(T::ItemName),
     Index(T::ItemName),
-    Query(Box<Query<T>>, bool),
+    Query(Box<SelectStatement<T>>, bool),
```

Comment:
I think we can now rename the `Query` to `Select` in both `Explainee` enums.

Reply:
Done.